src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -76,7 +76,7 @@ def validate_proximity_placement_group(cmd, namespace):
                                                            'proximityPlacementGroups', 'Microsoft.Compute')
 
     if not namespace.proximity_placement_group or not is_valid_resource_id(namespace.proximity_placement_group):
-        logger.debug('Proximity placement group is %s', namespace.proximity_placement_group)
+        logger.debug('namespace.proximity_placement_group is %s', namespace.proximity_placement_group)
```

Comment:
Why is this here?

Reply:
debug statement just in case the user enters an invalid proximity placement group. The previous lines try to parse the resource if and return None. Maybe I should actually raise a CLI error if this fails to parse...

What do you think?